src/sentry/tasks/derive_code_mappings.py

```diff
@@ -50,10 +50,11 @@ def derive_missing_codemappings(dry_run=False) -> None:
     queue="derive_code_mappings",
     max_retries=0,  # if we don't backfill it this time, we'll get it the next time
 )
-def derive_code_mappings(organization: Organization, dry_run=False) -> None:
+def derive_code_mappings(organization_id: int, dry_run=False) -> None:
     """
     Derive code mappings for an organization and save the derived code mappings.
     """
+    organization: Organization = Organization.objects.get(id=organization_id)
```

Comment:
Interesting that this is the preferred way, I would think the extra query is less optimal. TIL!